### DISCUSSION TITLE
Chart download image resolution

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1894] [General]
My customers want to have the image they downloaded by the chart download button in a higher resolution. They want to use it with PowerPoint for a presentation....

Where I can adjust the resolution?

![image](https://user-images.githubusercontent.com/94443931/206857114-7cbab3d4-25bd-4394-be3f-b3e56b411cc1.png)

**Comment by christoph-LSN** [comment]
It turned out that html2canvas has a scale option. https://html2canvas.hertzen.com/configuration

I added 

`scale: 2,` 

here: https://github.com/christoph-LSN/MT_Site/blob/a4861e125e665585104f53f37e1ede54c42f4bcf/_includes/assets/js/view/chartHelpers.js#L150

the following image is downloaded now

![indicator_1-1-1 (7)](https://user-images.githubusercontent.com/94443931/207062624-c1f8009f-2ac1-4e6b-a768-236e11dd45d9.png)

Perhaps it makes sence to make this optional with some kind of dialog with a radiobutton for standard or high resolution....

What do you think @brockfanning @LucyGwilliamAdmin ?

**Comment by brockfanning** [comment]
@christoph-LSN Making sure I understand right - the users need to expand the image to a higher size, but it is getting blurry? So they would like it to be downloaded already at a higher size?

**Comment by christoph-LSN** [comment]
@brockfanning exactly!
